tests/validation/tests/v3_api/test_node_template.py

```diff
@@ -0,0 +1,149 @@
+import os
+import pytest
+from rancher import ApiError
+from .common import *  # NOQA
+from .test_rbac import create_user
+
+DO_ACCESSKEY = os.environ.get('DO_ACCESSKEY', "None")
+
+user_token = {"stduser_with_createrketemplate_role": {"user": None,
+                                                      "token": None},
+              "standard_user": {"user": None, "token": None}}
+
+@pytest.fixture(scope='module', autouse="True")
+def setup(request):
+
+    client = get_admin_client()
+
+    # create users
+    user_token["stduser_with_createrketemplate_role"]["user"], \
```

Comment:
when `test_rbac` is set to `true`, some users will be created. Could we reuse those instead of creating new ones?